icicle_v3/backend/cpu/include/cpu_ntt.h

```diff
@@ -0,0 +1,331 @@
+#include "icicle/ntt.h"
+#include "icicle/errors.h"
+#include "icicle/runtime.h"
+#include "icicle/utils/log.h"
+#include "icicle/fields/field_config.h"
+
+#include <vector>
+#include <algorithm>
+#include <iostream>
+#include <cmath>
+#include <cstdint>
+#include <memory>
+#include <mutex>
+
+using namespace field_config;
+using namespace icicle;
+
+namespace cpu_ntt_h {
```

Comment:
I personally would remove the _h since it's a little weird but you can leave it like that if you like it

Reply:
changed to "ntt_cpu"